custom/dhis2/payload_generators.py

```diff
@@ -67,12 +67,13 @@ def get_payload(self, repeat_record, form):
             # enroll in the nutrition assessment programme.
             logger.debug('DHIS2: Processing Register Child form')
             push_case(case, dhis2_api)
-            # We just need to enroll. No event to create
+            return  # We just need to enroll. No event to create
```

Comment:
Yes, it will. That's why I think we need some way to mark this as succeeded without trying to send an empty payload. This is where it would raise an IgnoreDocument exception, if that's the approach we decide to use.